src/api/addresses.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  BTC_LEGACY_DERIVATION,
+  BTC_SEGWIT_DERIVATION,
+  BTC_WRAPPED_SEGWIT_DERIVATION,
+  DEFAULT_ETH_DERIVATION,
+  HARDENED_OFFSET,
+} from '../constants';
+import { queue } from './utilities';
+
+export const fetchAddresses = async (
+  overrides?: GetAddressesRequestParams,
+): Promise<string[]> => {
+  return queue((client) =>
+    client
+      .getAddresses({
+        startPath: DEFAULT_ETH_DERIVATION,
+        n: 10,
+        ...overrides,
+      })
+      .then((addrs) => addrs.map((addr) => `${addr}`)),
+  );
+};
+
+export const fetchBtcLegacyAddresses = async (n = 10): Promise<string[]> => {
```

Comment:
All of these BTC address fetchers are assuming a starting path at index 0, and do not use the same `...overrides` pattern. These things really need to be parametrizable because someone may want e.g. the 100th Ledger Live BTC address, which would be literally impossible with this function.

Reply:
I added the ability to pick the start index [235fd9e](https://github.com/GridPlus/gridplus-sdk/pull/525/commits/235fd9e7f44a6044fb7e5e6f850b64af23c5b4fc)